workflows/prognostic_c48_run/runtime/segmented_run/run.py

```diff
@@ -22,8 +22,23 @@ def cwd(path):
 def find(path: str):
     return glob.glob(os.path.join(path, "**"), recursive=True)
 
+def compose_simulation_command(mpi_launcher: str, nprocs: int): 
 
-def run_segment(config: dict, rundir: str):
+    command = ["-n " + str(nprocs), sys.executable, "-m", "mpi4py", runfile.absolute().as_posix()]
```

Comment:
This is the subtle change required for the tests to pass (this also makes things consistent with how they were written before for the `"mpirun"` case):
```suggestion
    command = ["-n", str(nprocs), sys.executable, "-m", "mpi4py", runfile.absolute().as_posix()]
```
See the note regarding "how to break a shell command into a sequence of arguments" in the [`subprocess.Popen` documentation](https://docs.python.org/3/library/subprocess.html#subprocess.Popen) for more details.  The documentation admits this "may not be obvious!"